app/views/schedules/_root_fields.erb

```diff
@@ -0,0 +1,16 @@
+<%= f.label :name %>
+<%= f.text_field :name, class: 'form-control' %>
+
+<h3>
+  Tasks
+</h3>
+<%= f.label :root_task_id, 'Root Task' %>
+<%= f.select :root_task_id,
+             Task.where(schedule_id: f.object.id).map { |t| ["##{t.id}", t.id] },
+             {include_blank: true, selected: (f.object.nil? || f.object.root_task.nil? ? '' : f.object.root_task.id)},
+             class: 'form-control' %>
+
+<div id="tasks" class="form-group">
+  <% # I want to just render the blank new Task I created in the controller, but I think this is failing because the schuled hasn't saved yet so it still thinks this is nil? %>
+    <%= render 'task_fields', f: f %>
```

Comment:
I'm getting an undefined method error when I try to create a new Schedule. It looks like the task_fields partial has a field for Type and Schedule doesn't have a Type. I think if you aren't calling it via Cocoon you might have to pass the root_task object directly into the partial.